components/shop/hero.module.css

```diff
@@ -0,0 +1,149 @@
+.shopHero {
+  width: 70%;
+  min-height: 64vh;
+  margin: 0 auto 4rem;
+  padding: 4rem 0 0 0;
+  display: flex;
+  flex-flow: wrap row;
+  align-items: stretch;
+  justify-content: center;
+  gap: 2rem;
+}
+
+.shopHero * {
+  -webkit-tap-highlight-color: transparent;
+}
+
+.content,
+.videoContainer {
+  flex-basis: 0;
+  flex-grow: 1;
+  flex-shrink: 1;
+  display: flex;
+  flex-flow: wrap column;
+  align-items: flex-start;
+  justify-content: flex-start;
+  gap: 1rem;
+}
+
+.title {
+  font-family: 'Abril Text';
+  font-size: 3.75rem;
+  font-weight: 700;
+  color: hsl(115, 100%, 75%);
+}
+
+.text {
+  width: 80%;
+  font-size: 1.25rem;
+  color: hsl(0, 0%, 95%);
+}
+
+.cta {
+  margin: 1rem 0 0 0;
+  padding: 0.9375rem 1.25rem;
+  position: relative;
+  overflow: hidden;
+  display: flex;
+  gap: 0.5rem;
+  align-items: center;
+  border-radius: 0.3125rem;
+  border: none;
+  cursor: pointer;
+  text-decoration: none;
+  font-size: 1rem;
+  font-weight: bold;
+  color: hsl(231, 15%, 18%);
+  background-color: transparent;
+  transition: none;
+  z-index: 1;
+}
+
+.cta * {
+  pointer-events: none;
+}
+
+.cta span,
+.cta svg {
+  z-index: 2;
+}
+
+.cta svg {
+  width: 1.25rem;
+  height: 1.25rem;
+  stroke: currentColor;
+  stroke-width: 1;
+  stroke-linecap: round;
+  stroke-linejoin: round;
+  fill: hsl(231, 15%, 18%);
+  transition-property: transform;
+  transition-duration: 0.16s;
+  transition-timing-function: ease-in-out;
+}
+
+.cta::before {
+  content: '';
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  inset: 0;
+  background-color: hsl(115, 100%, 75%);
+  transition-property: filter;
+  transition-duration: 0.16s;
+  transition-timing-function: ease-in-out;
+  z-index: 1;
+}
+
+.cta:focus::before,
+.cta:hover::before {
+  filter: brightness(1.16);
+}
```

Comment:
No need for a ::before selector, this seems like an overcomplicated solution to a simple button.